app/views/movies/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @movie.title %> Show Page </h1>
+<p><%=  @movie.creation_year%></p>
+<p><%=  @movie.genre%></p>
+<p><%= @movie.actors.ordered_by_age.join(", ") %></p>
+<p>Average Age: <%= @movie.actors.average_age %>
+
+<% @movie.actors.each do |actor| %>
+  <p><%= actor.name %></p>
+  <p><%= actor.age %></p>
+<% end %>
+
+<h4>Add an Actor Form</h4>
+<%= form_with url: "/movies/#{@movie.id}", method: :post, data: {turbo: false} do |form| %>
```

Comment:
Looks great as a POST request, but a slightly more RESTful path could be "movies/:id/actors" to indicate that you are creating an association between movies *and actors* rather than just including the actor resource in the path